apps/condo/index.js

```diff
@@ -231,23 +226,14 @@ module.exports = {
                     },
                 })
             if (!isSenderValid) {
-                res.cookie('sender', JSON.stringify({ fingerprint: cookies['userId'] || makeId(12), dv: 1 }))
-                res.cookie('dv', 1)
+                const fingerprint = cookies['userId'] || makeId(12)
+                res.cookie('sender', JSON.stringify({ fingerprint, dv: 1 }), { maxAge: 1707195600 })
```

Comment:
maybe add a constant for maxgAge?

Reply:
I don’t think in that case it’s increase the readability.

Reply:
Seems like a random number right now :)

Reply:
It's really true %) I've take it from https://developer.chrome.com/blog/cookie-max-age-expires/